openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py

```diff
@@ -25,118 +27,51 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
-            courseenrollment__is_active=True,
-        ).values_list('id', flat=True).distinct()
-
-    def check_users_for_application_update(self, users_to_be_checked_for_update, prereq_course_groups):
+    def check_users_for_application_update(self, users_to_be_checked_for_update):
         """
-        Check check passed prereq courses for all filtered users
+        Check user has passed prereq courses and update application hub flag
         Args:
             users_to_be_checked_for_update: List of user ids
-            prereq_course_groups: List of course groups
         Returns:
             None
         """
         users = User.objects.filter(id__in=users_to_be_checked_for_update)
         for user in users:
-            self.check_passed_prereq_courses_and_update_application_hub(user, prereq_course_groups)
-
-    def check_passed_prereq_courses_and_update_application_hub(self, user, prereq_course_groups):
-        """
-        Check passed prereq courses for a particular user and update application hub if all prereq courses are passed
-        Args:
-            user: User object
-            prereq_course_groups: List of course groups
-        Returns:
-            None
-        """
-        for prereq_course_group in prereq_course_groups:
-            if self.is_user_failed_in_course_group(user, prereq_course_group):
-                logger.info('{username} has not yet passed all the pre-reqs'.format(username=user.username))
-                break
-        else:
-            self.update_application_hub(user)
-
-    def is_user_failed_in_course_group(self, user, prereq_course_group):
-        """
-        Checks if user is failed in the given group of courses. User is considered passed if he has passed any one of
-        the courses from course group
-        Args:
-            user: User object
-            prereq_course_group: PrerequisiteCourseGroup model object
-        Returns:
-            boolean, True if the course is failed otherwise False
-        """
-        course_keys = prereq_course_group.open_multilingual_course_keys()
-
-        for course_key in course_keys:
-            course_grade = CourseGradeFactory().read(user, course_key=course_key)
-            course_passed = course_grade and course_grade.passed
-
-            if course_passed:
-                return False
-
-        return True
+            courses = MultilingualCourseGroup.objects.get_user_program_prereq_courses(user)
+            if check_user_passed_given_courses(user, courses):
+                self.update_application_hub(user)
 
     def update_application_hub(self, user):
         """
-        Update flag in application hub model for particular user, on successfully completing prereq courses
+        Update flag in application hub model for user, on successfully completing program prereq courses
         Args:
-            user: User object
+            user (User): User object
         Returns:
             None
         """
```

Comment:
Lets follow the convention and add spaces between function definition, Args and Returns